src/components/CarsDetails.tsx

```diff
@@ -0,0 +1,122 @@
+import { useParams, useNavigate } from 'react-router-dom'
+import { fetchWithAuth } from '../utils/api'
+import { useEffect, useState } from 'react'
+import CarIcon from '../assets/CarIcon'
+import Attention from '../assets/Attention.png'
+import licenseplate from '../assets/LicensePlate.png'
+import HorseIcon from '../assets/HorseIcon'
+import { ChevronBackIcon } from '../assets/ChevronBackIcon'
+import FuelIcon from '../assets/FuelIcon'
+import ProfileIcon from '../assets/ProfileIcon'
+
+interface Car {
+  id: string
+  name: string
+  ownerId: string
+  model: string
+  licensePlate: string
+  horsepower: string
+  fuelType: string
+  state: string
+  carTypeId: string
+}
+
+interface CarType {
+  id: string
+  imageUrl: string
+  name: string
+}
+
+export default function CarsDetails() {
+  const { id } = useParams<{ id: string }>()
+  const navigate = useNavigate()
+  const [car, setCar] = useState<Car | null>(null)
+  const [carType, setCarType] = useState<CarType | null>(null)
+  const [loading, setLoading] = useState(true)
+
+  useEffect(() => {
+    async function fetchCarDetails() {
+      try {
+        const carData = await fetchWithAuth(`/cars/${id}`)
+        setCar(carData)
+
+        if (carData.carTypeId) {
+          const typeData = await fetchWithAuth(`/car-types/${carData.carTypeId}`)
+          setCarType(typeData)
+        }
+      } catch (error) {
+        console.error('Error fetching car details:', error)
+      } finally {
+        setLoading(false)
+      }
+    }
+
+    fetchCarDetails()
+  }, [id])
+
+  if (loading) return <p className="text-center mt-10 text-white">Loading car details...</p>
+  if (!car) return <p className="text-center mt-10 text-gray-300">Car not found.</p>
+
+  return (
+    <div className="flex min-h-screen flex-col items-center bg-gradient-to-b to-sky-700 text-white">
+      <div className="mt-6 flex w-full items-center gap-2 px-6">
+        <button onClick={() => navigate(-1)}>
+          <ChevronBackIcon />
+        </button>
+        <h1 className="mx-auto mt-11 text-2xl font-semibold">DETAILS</h1>
```

Comment:
```suggestion
        <h1 className="mx-auto mt-11 text-2xl font-semibold font-lora">DETAILS</h1>
```
Great job here, but I think it'll be a good idea to add the font that was used in the Figma design.